src/TableV8/styled.tsx

```diff
@@ -44,10 +45,37 @@ export const TABLE_CLASSES = {
 const COLORS = {
   BACKGROUND_HIGHLIGHT: colors.grey_3,
   BACKGROUND_SECONDARY: colors.grey_4,
+  BACKGROUND_SELECTED: colors.secondary_4,
   BACKGROUND: colors.white,
   BORDER: colors.grey_2,
 };
 
+const tableCellHeaderStyle = css`
```

Comment:
moved this CSS from classes to CSS strings so that CSS props from platform-UI will stack properly with the CSS in this file